src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -110,6 +138,7 @@ public static Tag parseTag(String tag) throws ParseException {
         return new Tag(trimmedTag);
     }
 
+    //@@author
```

Comment:
Is this line intended?

Reply:
Yeah, an empty //@@author tag is needed to indicate that the following methods aren't authored by me.

Reply:
Ic, thanks!